docs/docs/tutorial/part-eight/index.md

```diff
@@ -26,7 +26,7 @@ Lighthouse is included in Chrome DevTools. Running its audit -- and then address
 
 Try it out!
 
-First, you need to create a production build of your Gatsby site. The Gatsby development server is optimized for making development fast; But the site that it generates, while closely resembling a production version of the site, isn't as optimized.
+First, you need to create a production build of your Gatsby site. The Gatsby development server is optimized for making development fast, but the site that it generates -- while closely resembling a production version of the site -- isn't as optimized.
```

Comment:
I agree with you on the capital B, but the are you have replaced with `--`'s are perfectly fine as an [embedded clause](https://www.theschoolrun.com/what-is-an-embedded-clause#:~:text=In%20other%20words%2C%20an%20embedded,information%20and%20enhance%20the%20sentence.).

```suggestion
First, you need to create a production build of your Gatsby site. The Gatsby development server is optimized for making development fast but the site that it generates, while closely resembling a production version of the site, isn't as optimized.
```